web/packages/teleport/src/Account/Account.test.tsx

```diff
@@ -189,6 +192,14 @@ test('password change', async () => {
   const ctx = createTeleportContext();
   ctx.storeUser.setState({ passwordState: PasswordState.PASSWORD_STATE_UNSET });
   jest.spyOn(ctx.mfaService, 'fetchDevices').mockResolvedValue([]);
+  jest.spyOn(auth, 'getMfaChallenge').mockResolvedValue({
+    webauthnPublicKey: {} as PublicKeyCredentialRequestOptions,
```

Comment:
Shouldn't this be null? I think if its not returned during the challenge its null

Reply:
We want it to be returned during the challenge here, so it needs to be non-null

Reply:
Oh duh, that makes sense

Reply:
Eh, it's not necessary for this test which just cares about option selection, some other tests mock it. Maybe we'll need to add in the future though.